packages/aft/lib/src/models.dart

```diff
@@ -80,11 +140,20 @@ class PackageInfo
             falsePositiveExamples.contains(name));
   }
 
+  /// Skip package checks for Flutter packages when running in CI without
+  /// Flutter, which may happen when testing Dart-only packages or specific
+  /// Dart versions.
```

Comment:
I don't think I follow this. Would this be a dart-only package that has flutter listed as a dependency for some reason?

Reply:
This is a roundabout way of saying "don't try to run `flutter pub upgrade` when `flutter` is not available"